src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordCommon.cs

```diff
@@ -399,15 +433,17 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
             // this variable doubles as a flag to see if we need to generate doc comments at all, as
             // if it is still null, we found no meaningful doc comments anywhere
             var exteriorTrivia = GetExteriorTrivia(typeDeclaration) ??
-                properties.SelectAsArray(GetExteriorTrivia).FirstOrDefault(t => t != null);
+                propertyResults.SelectAsArray(result => GetExteriorTrivia(result.Declaration)).
+                    FirstOrDefault(t => t != null);
```

Comment:
Hmm, not something really changed, but why the SelectAsArray if you're just going to do a FirstOrDefault()? Will that just process more only to ignore it?

Reply:
(this was marked as resolved but not sure I'm seeing a resolution)